examples/functional_tests.py

```diff
@@ -0,0 +1,94 @@
+import sys
+from openrazer.client import DeviceManager
+from openrazer.client import constants as razer_constants
+
```

Comment:
A quick "about" section would be helpful. Things like use cases for this tool and expected invocation.